modules/tinymce/src/plugins/fullscreen/test/ts/browser/FullScreenPluginTest.ts

```diff
@@ -22,51 +43,87 @@ export const cWaitForDialog = (ariaLabel) =>
           throw new Error('Dialog did not have an aria-labelledby');
         }
       })
-    ]),
-    Guard.addLogging('Looking for dialog with an aria-label: ' + ariaLabel)
+    ])
+  );
+
+const cAssertHtmlAndBodyState = (label: string, shouldExist: boolean) => {
+  const selector = shouldExist ? 'root:.tox-fullscreen' : 'root::not(.tox-fullscreen)';
+  return Chain.label(
+    `${label}: Body and Html should ${shouldExist ? '' : 'not'} have "tox-fullscreen" class`,
+    Chain.fromIsolatedChains([
+      Chain.inject(SugarBody.body()),
+      UiFinder.cFindIn(selector),
+      Chain.inject(SugarElement.fromDom(document.documentElement)),
+      UiFinder.cFindIn(selector)
+    ])
+  );
+};
+
+const cAsssertEditorContainerAndSinkState = (label: string, shouldExist: boolean): Chain<Editor, Editor> =>
+  Chain.label(
+    `${label}: Editor container and sink should ${shouldExist ? '' : 'not'} have "tox-fullscreen" class and z-index`,
```

Comment:
An _ultra_ minor thing, but this will result in a double space, maybe:

```suggestion
    `${label}: Editor container and sink should ${shouldExist ? 'have' : 'not have'} "tox-fullscreen" class and z-index`,
```

or


```suggestion
    `${label}: Editor container and sink should${shouldExist ? ' ' : ' not '}have "tox-fullscreen" class and z-index`,
```

Reply:
I don't mind if this isn't changed, just noticed it